samples/ppm.cc

```diff
@@ -52,10 +52,10 @@ std::pair<amber::Result, std::string> ConvertToPPM(
   // Write PPM data
   for (amber::Value value : values) {
     const uint32_t pixel = value.AsUint32();
-    // We assume R8G8B8A8_UINT here:
-    image.push_back(byte0(pixel));  // R
+    // We assume B8G8R8A8_UNORM here:
+    image.push_back(byte2(pixel));  // R
```

Comment:
The comment is out of sync here, it should read BGRA

Reply:
Hah, good catch. Updated the top comment and missed all the inline ones.

Reply:
Actually, I think these are right. We're writting the R value here, not the B value. It just happens that R is in byte2.

Does that make sense?

Reply:
Woops my bad, I got confused, the comments are correct indeed!